api/envoy/admin/v2alpha/mutex_stats.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package envoy.admin.v2alpha;
+
+// [#protodoc-title: Mutex Stats]
+
+// Proto representation of the statistics collected upon absl::Mutex contention, if Envoy is run
+// under --enable-mutex-tracing. For more information, see the `absl::Mutex`
+// [docs](https://abseil.io/about/design/mutex#extra-features).
+message MutexStats {
+
+  // The number of individual mutex contentions which have occurred since startup.
+  uint64 num_contentions = 1;
+
+  // The length of the current contention wait cycle.
```

Comment:
maybe just a note about what a 'cycle' means?

Reply:
Done in the NB in 677c64a.